sample/F8/tracks

```diff
@@ -0,0 +1,202 @@
+[
+   {
+      "id":1,
+      "name":"ADS & MONETIZATION",
+      "color":"#BF532E",
+      "description":null,
+      "location":"C",
```

Comment:
@niranjan94 Those fields are still listed in the API docs, shouldn't they be removed then?